sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -27,6 +29,8 @@ import org.apache.spark.sql.catalyst.util.ArrayData
 import org.apache.spark.sql.types._
 import org.apache.spark.unsafe.types.{ByteArray, UTF8String}
 
+import scala.util.control.NonFatal
```

Comment:
Oh, we have import ordering. You should move `NonFatal` like the following. Blank line is needed.
```
import java.util.Date

import scala.util.control.NonFatal

import org.apache.hadoop._

import org.apache.spark.sql._
```